readme.md

```diff
@@ -477,11 +477,11 @@ Make array a setoid:
 ```js
 Array.prototype.equals = (arr) => {
```

Comment:
I'll update it to whatever you choose. My personal preference is to _always_ include the parentheses, but your call!